chapters/meta-information.adoc

```diff
@@ -192,7 +191,7 @@ to and a unique a short identifier for the functional component itself:
 
```

Comment:
Line 186: 

> organizational changes (Zalando for example maintains an internal naming convention).

Just remove everything in () (and parenthesis itself)

Reply:
Should be fixed now